src/smokeTest/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.main.allow-bean-definition-overriding=true
```

Comment:
this should not be added in as a default, this means you've likely got a bug in your application.